Heading/Heading.module.less

```diff
@@ -15,6 +23,17 @@
 		text-transform: @agate-heading-text-transform;
 		font-variant-ligatures: @agate-heading-font-ligatures;
 
+		.backButton {
+			.bg {
+				border: none;
+				background-color: transparent;
+			}
```

Comment:
We have `backgroundOpacity="transparent"` prop instead of this.

Reply:
Yes, you are right. Thank you!